doc/api/test.md

```diff
@@ -578,8 +589,15 @@ test('spies on an object method', (t) => {
     add(a) {
       return this.value + a;
     },
+    error() {
+      throw new Error('error');
+    },
```

Comment:
This code sample is about mocking methods, I don't think `assert.throws` has anything to do with it.
```suggestion
```